website/docs/r/federated_settings_identity_provider.html.markdown

```diff
@@ -49,6 +49,7 @@ In addition to all arguments above, the following attributes are exported:
 ### FederatedSettingsIdentityProvider
 
 * `okta_idp_id` - Unique 20-hexadecimal digit string that identifies the IdP.
+* `id` - Unique 24-hexadecimal digit string that identifies the IdP.
```

Comment:
Is this accurate? From my understanding the id of the resource is not modified, and continues to be de the encoded value of the federation_settings_id and okta_idp_id.

Reply:
you are right, the terraform state id continues to be de the encoded value of the federation_settings_id and okta_idp_id. But once users start using the new `id`, terraform state id will be federation_settings_id and id encoded

Reply:
I do agree that now if the users import the resource the `id` value of the resource with be encoded (federation_settings_id + new format of identity provider id). However, my question is if this documentation of `id` which was added is accurate, because the id value will still be an encoded value which I doubt users will be able to use.